### DISCUSSION TITLE
ScriptReference/RenderTexture.Release

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#529] [Notes]
# ScriptReference/RenderTexture.Release

https://docs.unity3d.com/ScriptReference/RenderTexture.Release.html

**Comment by electronics45** [comment]
If your getting the error message:
Assertion failed on expression 'srcAttach < m_CurrentFramebuffer.colorCount && "We should always resolve only current RT"
When releasing your render texture, then remember to make sure your set RenderTexture.active to null, or preferably, restore it's previous value before you release it.
